ci

```diff
@@ -533,7 +533,16 @@ case "$1" in
     # Set up build environment
     nuget restore tools/Vale/src/packages.config -PackagesDirectory tools/FsLexYacc 
 
-    run_log_commit "scons.bat -j 4" "vale-build" "#vale-build" "project-everest/vale"
+    #since each build verifies x86, x64 and ARM, only verify on one build as takes quite a bit longer to verify
+    if [[ "$OSTYPE" == "linux-gnu" ]]; then
```

Comment:
This is correct but sadly won't work the day we have OSX builds. You can either flip the if-then-else into:

```
if [[ $OS == "Windows_NT" ]]; then
  if [[ "$Platform" ... ]]
    ...
  else
    ...
  endif
else
  scons -j 4 # works for any future non-windows platform (osx, linux, perhaps unix, who knows, etc.)
endif
```

or refine the test to be "linux-gnu" or the value of this environment variable on osx.